cmd/captain/config.go

```diff
@@ -149,7 +149,9 @@ func findInParentDir(fileName string) (string, error) {
 // InitConfig reads our configuration from the system.
 // Environment variables take precedence over a config file.
 // Flags take precedence over all other options.
-func InitConfig(cmd *cobra.Command, cliArgs *CliArgs) (cfg Config, err error) {
+func InitConfig(cmd *cobra.Command, cliArgs *CliArgs, args []string) (cfg Config, err error) {
+	cliArgs.RootCliArgs.positionalArgs = args
```

Comment:
Since there are only two places where we call `InitConfig`, why don't we make this assignment before passing `cliArgs`.

Keep in mind that `cliArgs` is a point and now `InitConfig` is mutating `cliArgs`, which is not apparent from reading the function signature or comment.

Reply:
(appreciate the nudge, thanks!_